README.md

```diff
@@ -512,9 +512,10 @@ Returns the response so other methods can be chained.
 *Tip: Use Deno's [`status`](https://deno.land/std/http/http_status.ts) constants to define the status code.*
 
 ```js
+import pogo from 'https://deno.land/x/pogo/main.js';
 import { Status as status } from 'https://deno.land/std/http/http_status.ts';
-const handler = (request) => {
-    return request.response.code(status.Teapot);
+const handler = (request:pogo.Request, h:pogo.Toolkit) => {
```

Comment:
These classes, `Request` and `Toolkit`, aren't actually exposed by `main.js`. So this doesn't work, does it?

Reply:
it does for me, but i am using TS and the library is not currently, so there may be an interfacing issue. in any case i prefer to do this over :any

Reply:
Some kind of TypeScript magic, fascinating. Alright, I'll accept this. When the TypeScript conversion in #14 is merged, do you think it would be reasonable to remove these and use the contextually inferred types? I'm not super familiar with TS conventions yet.

I'd like to keep the docs simple and approachable, maybe even limited to plain JS syntax in most cases (aside from specific examples of TypeScript usage). But I acknowledge that since type inference doesn't work well at the moment, adding types here makes sense until Pogo is written in TypeScript.

Reply:
absolutely, simple docs are a big plus as long as there are enough concrete examples. my initial problem was that i did not know the signature of the handler method and therefore had not much of an idea what to do. concrete type mentions can help with that as it points you into a direction.

Reply:
Thanks. One last question: why did you switch from `request.response` to `h.response()`? They're both fine, to be clear. The latter is a bit more functionally pure, which is nice, but since it introduces an additional concept, I figured the former might be more approachable. I'm merging this as-is since it's not super important, but curious to hear your thought process.